http/src/main/resources/init-resources/gce-init.sh

```diff
@@ -209,13 +215,15 @@ log 'Formatting and mounting persistent disk...'
 # Fix this to `sdb`. We've never seen a device name that's not `sdb`,
 # Altho you some images, this cmd $(lsblk -o name,serial | grep 'user-disk' | awk '{print $1}')
 # can be used to find device name, this doesn't work for COS images
-export DISK_DEVICE_ID='sdb'
+USER_DISK_DEVICE_ID=$(lsblk -o name,serial | grep 'user-disk' | awk '{print $1}')
```

Comment:
`lsblk -o name,serial | grep 'user-disk` doesn't work all the time, hence giving it a default `sdb`...I saw once it wasn't `sdb`, but it was when I was using a wrong image, so not sure if that'll actually happen normally.